code/src/components/Task.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import { todos } from '../reducers/todos'
+import styled from 'styled-components'
+import { Trash } from '../icons/Trash'
+
+export const Task = (props) => {
+    const dispatch = useDispatch()
+    const handleCheckboxClick = () => {
```

Comment:
here you coudl destructure the props to not have to write props.item everywhere.
Either like this:
`const {id, name, complete} = props.item`
or when creating the Task component:
`export const Task = ( {id, name, complete} ) => { ... `